### PR TITLE
DASH-469 Add filters for admin

### PR DESCRIPTION
**Pull request by vmkarpenko**

### CODE REVIEW

**Review by orkhan-huseyn — APPROVED** [review]

File: client/src/api/admin.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import {
   UsersDetailsResponse,
   UsersListResponse,
 } from './responses';
-import { RefProfile } from '../types';
+import { AnyObject, RefProfile } from '../types';
```
Comment (orkhan-huseyn) [inline client/src/api/admin.ts:18]: Can someone explain to me why is it okay to use `AnyObject` but not `any`?
In the context of our project. @vmkarpenko @trukhilio 

image

Comment (trukhilio) [reply client/src/api/admin.ts:18]: @orkhan-huseyn We have ts rule that we shouldn't use `any` type. We have warnings for that. But in some cases it's too hard or to set correct type we doesn't know it. In this case we have one place where we ignore this rule and don't have a bunch of warnings about `any` type. But we try to use it when there is no any way to set proper type.

File: client/src/api/admin.ts [hunk 2]
```diff
@@ -52,8 +52,12 @@ export default class Admin extends Base {
     return this.apiClient.get(`admin/info/${id}`);
   }
 
-  ordersList(limit: number, offset: number): Promise<AdminOrdersListResponse> {
-    return this.apiClient.get('admin/orders', { limit, offset });
+  ordersList(
+    limit: number,
+    offset: number,
+    filters?: AnyObject,
```
Comment (trukhilio) [inline client/src/api/admin.ts:58]: I see you have type value for filters in `client/src/pages/AdminOrdersPage/AdminOrdersPage.tsx` - `Partial<OrderDetails>`. Could you use it here too?

**Review by trukhilio — APPROVED** [review]
Check comments

**Pull request merged**